src/e2e-test/java/io/cdap/plugin/sfmcsource/stepsdesign/DesignTimeSteps.java

```diff
@@ -16,11 +16,59 @@
 
 package io.cdap.plugin.sfmcsource.stepsdesign;
 
+import com.exacttarget.fuelsdk.ETSdkException;
 import io.cdap.e2e.utils.CdfHelper;
+import io.cdap.plugin.sfmcsource.actions.SfmcSourcePropertiesPageActions;
+import io.cdap.plugin.utils.enums.Sobjects;
+import io.cucumber.datatable.DataTable;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 
 /**
  * Represents Salesforce Marketing Cloud - Source plugin - Properties page - steps.
  */
 public class DesignTimeSteps implements CdfHelper {
 
+  List<Sobjects> objectsList = new ArrayList<>();
```

Comment:
can we move this into selectObjectsInObjectsList()? looks like it's not used anywhere else

Reply:
yes we can move it, moved into selectObjectsInObjectsList(). (https://github.com/data-integrations/salesforce-marketing/pull/34/commits/9bff48a06d63e875c4efa2c7ce37fe85eca8f3a3#diff-9dd183fb9ead04a9d5eb6e195c3c7bf9e2f948ec1301777bc9fedae51ceae407)